tldr_bot.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import requests
+from flask import Flask, request, make_response
+
+app = Flask(__name__)
+
+def post_comment(issue_id, body):
+    url     = '{}/repos/{}/issues/{}/comments'.format(GITHUB_API_URL, REPO_SLUG, issue_id)
+    headers = {'Authorization': 'token ' + BOT_TOKEN}
+    data    = {'body': body}
+
+    return requests.post(url, json=data, headers=headers)
+
+
+def check_already_commented(issue_id):
+    url  = '{}/repos/{}/issues/{}/comments'.format(GITHUB_API_URL, REPO_SLUG, issue_id)
+    resp = requests.get(url)
+
+    if resp.status_code != 200:
+        # GitHub API error, let's say True in this case to avoid problems.
+        return True
+
+    comments = resp.json()
+    print(comments)
+    return any(c['user']['login'] == 'tldr-bot' for c in comments)
+
+
+@app.route('/', methods=['POST']) # old route, to be removed
+@app.route('/comment', methods=['POST'])
+@app.route('/comment/once', methods=['POST'])
+def comment():
+    data = request.get_json()
+
+    if data is None:
+        return make_response('Invalid JSON or inappropriate Content-Type.', 400)
+
+    pr_id        = data.get('pr_id')
```

Comment:
Is this the standard Pythonic way to put additional space to match up with `=` sign ? I don't remember doing this a few years back.

Reply:
Don't know it it's pythonic or not, it's just a personal habit. I think it reads clearer. I can remove it if you want.

Reply:
Unless there is an automatic code formatter which does this (like gofmt for Go), I would lean towards using single-space between the `=` sign. We should always try to keep it consistent with other Python codebases so that new devs aren't surprised when they see it.

Reply:
Just checked PEP 8, and not aligning is the recommended choice. Fixed.